conan/api/subapi/download.py

```diff
@@ -1,7 +1,7 @@
 from conan.api.model import Remote
 from conan.api.subapi import api_method
-from conans.cli.conan_app import ConanApp
-from conans.cli.output import ConanOutput
+from conan.cli.command import ConanOutput
```

Comment:
But then the ``ConanOutput`` is still inside ``cli``. Isn't this ``from conan.cli`` import from the ``api`` that we want to avoid? It is true that the ``ConanApp``, maybe it belong below ``api``?